bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -3,9 +3,18 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
     this.sb = sandbox;
```

Comment:
Looks like instance is only used for getLocalization() calls in this component? Maybe just send the localization as parameter instead of the instance to make it more testable.